ApiCatalogoProducao/Services/FilmeService.cs

```diff
@@ -30,17 +30,17 @@ public List<Filme> ListarFilmes()
             return filmeRepository.ListarFilmes();
         }
 
-        public Filme PublicarFilme(Filme filme)
-        {
-            //Adicionar data de publicacao como data atual
-            //
+        //public Filme PublicarFilme(Filme filme)
```

Comment:
Favor não comentar o código. Quando for assim favor excluir :)

Reply:
Feito